pkg/securityhubcollector/security_hub_collector.go

```diff
@@ -237,62 +290,44 @@ func (h *HubCollector) ConvertFindingToRows(finding *securityhub.AwsSecurityFind
 	}
 
 	return output
-
 }
 
-// WriteFindingsToOutput - takes a list of security
-func (h *HubCollector) WriteFindingsToOutput(findings []*securityhub.AwsSecurityFinding, writeHeaders bool) (err error) {
-	var f *os.File
-	if writeHeaders {
-		// Try to create the output file we got from the collector object
-		f, err = os.Create(h.Outfile)
-		if err != nil {
-			return err
-		}
-	} else {
-		f, err = os.OpenFile(h.Outfile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
-		if err != nil {
-			return err
-		}
+// WriteHeadersToOutput - writes headers to the output CSV file
+func (h *HubCollector) WriteHeadersToOutput() error {
```

Comment:
You could make this not-exported and do it in `Init`. Unless there's some reason you could conceivably want to not write the headers, or to write them again at the end.